src/indexPage/endpoint.js

```diff
@@ -0,0 +1,65 @@
+import { get as getStorage, set as setStorage } from 'binary-common-utils/lib/storageManager';
+import { LiveApi } from 'binary-live-api';
+import { getWebSocketURL, getDefaultEndpoint, getAppIdFallback } from '../common/appId';
+
+export default function endpoint() {
+    if (!document.location.href.match(/endpoint\.html/)) return false;
```

Comment:
I can imagine that moving this into a react component can help with the functionality. (We can move the `endpoint.html` to `#endpoint` route then.